bindata/network/openshift-sdn/sdn.yaml

```diff
@@ -47,6 +47,9 @@ spec:
           #!/bin/bash
           set -euo pipefail
 
+          # tee all output to the host's journal
+          exec 1> >(logger -p user.info --size 4096 -s -t openshift-sdn/${K8S_POD_NAME}) 2>&1
```

Comment:
This just seems terrible, but there doesn't seem to be any more obvious way of either (a) getting glog to log to syslog instead of stderr or (b) getting kubernetes to output pod logs to syslog.

Reply:
years ago, as I was on the rkt / rktnetes project, we were trying to get structured logging + journald shipping in to k8s. Everyone said "no, docker log is good enough, honestly" and the project went nowhere.

Yup.